### DISCUSSION TITLE
Is there a way to add possibly_sensitive to a tweet?

### DISCUSSION THREAD

**Discussion by ronaldpk** [danieldevine/bird-elephant#54] [Q&A] [answered]
I'd like to add the ``possibly_sensitive`` property to some posts. Tried by simply adding it to the tweet object:

```php
 // compose the tweet
$tweet = (new \Coderjerk\BirdElephant\Compose\Tweet)->text('one more test...');
$tweet->possibly_sensitive = true;
// send the tweet
$response = $oApi->tweets()->tweet($tweet);
```
But that throws an error 400, bad request: "possibly_sensitive: is not defined in the schema".

Anyone have a suggestion?

**Comment by danieldevine** [comment, marked as answer]
I don't think there's a way to do this using the v2 api I'm afraid.

**Comment by ronaldpk** [comment]
OK, thanks for the reply. I thought v1 had such an option but it seems I misread the docs.
